include/dicom.h

```diff
@@ -5,6 +5,21 @@
 #ifndef DCM_INCLUDED
 #define DCM_INCLUDED
 
+#if defined(_WIN32) && !defined(__GNUC__)
+#include <BaseTsd.h>
+typedef SSIZE_T ssize_t;
+#endif
+
+#ifdef _WIN32
+#ifdef BUILDING_LIBDICOM
+#define DICOM_EXTERN __declspec(dllexport) extern
```

Comment:
Do we want to use `DCM_` rather than `DICOM_` or does this have to match the name of the library?